src/core/tests/Test_PatchAssessor.py

```diff
@@ -181,9 +180,18 @@ def test_raise_if_min_python_version_not_met(self):
             self.runtime.patch_assessor.start_assessment()
         self.assertEqual(str(context.exception), Constants.PYTHON_NOT_COMPATIBLE_ERROR_MSG.format(sys.version_info))
 
+    def test_raise_add_error_to_status(self):
```

Comment:
This should be in Test_StatusHandler.py

Reply:
I don't see why this needs to be moved to statushandler, statushandler capture the output of operation, this test is asserting the exception at time operation starts and the function patch_assessor.start_assessment is a public method in patchassessor, so I believe it's better to have it corresponding to class file

Reply:
In that case, change the test function name to reflect the scenario. For eg: test_patch assessment_throws_exception()
test_raise_add_error_to_status() indicates you are testing **add_error_to_status** which is a functionality of StatusHandler

Reply:
done